src/main/java/gyro/google/compute/InstanceAccessConfig.java

```diff
@@ -0,0 +1,122 @@
+import gyro.core.resource.Diffable;
+import gyro.google.Copyable;
+
+public class InstanceAccessConfig extends Diffable implements Copyable<AccessConfig> {
+
+    private String name;
+    private String natIp;
+    private String networkTier;
+    private String publicPtrDomainName;
+    private Boolean setPublicPtr;
+    private String type;
+
+    /**
+     * Name of this access configuration. Default and recommended name is ``External NAT``, but can be any arbitrary string.
+     */
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    /**
+     * An unused static external IP address available to the project. Leave this field undefined to use an IP from a shared ephemeral IP address pool. If you specify a static external IP address, it must live in the same region as the zone of the instance.
+     */
+    public String getNatIp() {
+        return natIp;
+    }
+
+    public void setNatIp(String natIp) {
+        this.natIp = natIp;
+    }
+
+    /**
+     * Signifies the networking tier used for configuring this access configuration. Valid values are ``PREMIUM`` or ``STANDARD``. If specified without a valid external IP address, an ephemeral IP will be created with this networkTier. If a valid external IP address is specified, it must match that of the networkTier associated with the Address resource owning that IP.
+     */
+    public String getNetworkTier() {
+        return networkTier;
+    }
+
+    public void setNetworkTier(String networkTier) {
+        this.networkTier = networkTier;
+    }
+
+    /**
+     * The DNS domain name for the public PTR record. Can only be set if the ``setPublicPtr`` field is set to ``true``.
+     */
+    public String getPublicPtrDomainName() {
+        return publicPtrDomainName;
+    }
+
+    public void setPublicPtrDomainName(String publicPtrDomainName) {
+        this.publicPtrDomainName = publicPtrDomainName;
+    }
+
+    /**
+     * if ``true`` a public DNS 'PTR' record should be created to map the external IP address of the instance to a DNS domain name.
+     */
+    public Boolean getSetPublicPtr() {
+        return setPublicPtr;
+    }
+
+    public void setSetPublicPtr(Boolean setPublicPtr) {
+        this.setPublicPtr = setPublicPtr;
+    }
+
+    /**
+     * The type of configuration. The default and only valid value is ``ONE_TO_ONE_NAT``.
+     */
+    public String getType() {
+        return type;
+    }
+
+    public void setType(String type) {
+        this.type = type;
+    }
+
+    @Override
+    public String primaryKey() {
+        return "";
+    }
+
+    @Override
+    public void copyFrom(AccessConfig model) {
+         setName(model.getName());
+         setNatIp(model.getNatIP());
+         setNetworkTier(model.getNetworkTier());
+         setPublicPtrDomainName(model.getPublicPtrDomainName());
+         setSetPublicPtr(model.getSetPublicPtr());
+         setType(model.getType());
```

Comment:
Indentation is off here, should be 8 rather than 9.

Reply:
Done.